one-ui/ui/src/pages/Bench.test.tsx

```diff
@@ -1,9 +1,185 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import Bench from './Bench';
-
-describe('Bench component', () => {
-  it('should render correctly', () => {
-    shallow(<Bench />);
-  });
-});
\ No newline at end of file
+import { render, fireEvent, waitForElement, cleanup } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import axiosMock from 'axios'
+import  Bench from '../pages/Bench';
+import {IRolesContextInterface, RolesContext} from '../util/roles';
+
+jest.mock('axios');
+
+class MockRolesService implements IRolesContextInterface{
```

Comment:
MockRolesService is something that can be reused by other "pages" in test, so we should probably create a separate module. There is a __mocks__ folder under src/api to store mock functions